cocos/core/assets/index.ts

```diff
@@ -55,3 +55,4 @@ export { Mesh, RenderingSubMesh } from './mesh';
 export { Skeleton } from './skeleton';
 export { RenderTexture } from './render-texture';
 import './deprecation';
+export { ParticleAsset } from './particle-asset';
```

Comment:
Can we put this file to `physics-2d` module?

Reply:
> Can we put this file to `physics-2d` module?

Moved to particle-2d.